crates/bevy_text/src/pipeline.rs

```diff
@@ -114,4 +117,140 @@ impl TextPipeline {
 
         Ok(TextLayoutInfo { glyphs, size })
     }
+
+    pub fn create_text_measure(
+        &mut self,
+        fonts: &Assets<Font>,
+        sections: &[TextSection],
+        scale_factor: f64,
+        text_alignment: TextAlignment,
+        linebreak_behaviour: BreakLineOn,
+    ) -> Result<TextMeasureInfo, TextError> {
+        let mut auto_fonts = Vec::new();
+        let mut scaled_fonts = Vec::new();
```

Comment:
those two vecs could be created with the correct size to avoid resizing down the road